tools/publish-verification-info.sh

```diff
@@ -1,34 +1,58 @@
 #!/bin/bash
 
-verification_dir="$1"
-img="$1/verification.raw"
+img="$1"
 ip="$2"
 
-if [ "$verification_dir" == "" ] || [ "$ip" == "" ]; then
-  echo "Usage ./publish_verification_info.sh <verification_dir> <server_ip>"
+if [ "$img" == "" ] || [ "$ip" == "" ]; then
+  echo "Usage ./publish_verification_info.sh <verification_img> <server_ip>"
   exit
 fi
 
-size=$(fdisk -l "$img" | grep raw5 | awk '{print $2}')
-mkdir /tmp/verification_mnt
-sudo mount -o offset=$((size*512)) "$img" /tmp/verification_mnt
 
-dirname=$(cat /tmp/verification_mnt/*/hardwaremodel.txt | grep "Product Name:" | awk '{ for (i = 3; i <= NF; i++) printf "%s", $i; printf "\n" }' | tr ',' '-' | tr '(' '-' | tr ')' '-' | tr '+' '-')
-eve_version=$(cat "${verification_dir}/verification/eve_version")
+mountDir="/tmp/verification_mnt"
+if [[ "$OSTYPE" == "darwin"* ]]; then #macOS
```

Comment:
Why double square brackets?

Reply:
Same questions for the tests below.

Reply:
The double square brackets allow comparison and boolean operators. I can make the equivalent statement with single brackets, but it is more convenient the way I did. If you think I should only use single brackets I will change it.

Reply:
I must be missing something. Everywhere else in this script and other shell scripts we use single brackets for tests/comparisons.

Reply:
I see; but we clearly don't need this non-posix construct for these simple comparisons.